about.php

```diff
@@ -0,0 +1,330 @@
+													<span>$125 night</span>
+												</div>
+											</div><!--news end-->
+										</div><!--news-widget end-->
+									</div><!--widget end-->
+								</div>
+								<div class="col-md-3">
+									<div class="widget">
+										<div class="widget-quick-links">
+											<h3 class="widget-title">Quick Links</h3>
+											<div class="qk-links">
+												<ul>
+													<li><a href="#" title="">Home</a></li>
+													<li><a href="#" title="">About Us</a></li>
+													<li><a href="#" title="">Suits</a></li>
+													<li><a href="#" title="">News</a></li>
+													<li><a href="#" title="">Contact Us</a></li>
+													<li><a href="#" title="">Bookisnt</a></li>
+													<li><a href="#" title="">Blogs</a></li>
+												</ul>
+												<ul>
+													<li><a href="#" title="">Activities</a></li>
+													<li><a href="#" title="">Gallery</a></li>
+													<li><a href="#" title="">Aminities</a></li>
+													<li><a href="#" title="">Single Room</a></li>
+													<li><a href="#" title="">Testimonials</a></li>
+													<li><a href="#" title="">Dinning</a></li>
+													<li><a href="#" title="">Offers</a></li>
+												</ul>
+											</div>
+										</div><!--widget-quick-links end-->
+									</div><!--widget-end-->
+								</div>
+								<div class="col-md-2 lst">
+									<div class="widget">
+										<div class="widget-tags">
+											<h3 class="widget-title">Tags</h3>
+											<ul>
+												<li><a href="#" title="">Booking</a></li>
+												<li><a href="#" title="">TV</a></li>
+												<li><a href="#" title="">Services</a></li>
+												<li><a href="#" title="">Room</a></li>
+												<li><a href="#" title="">Hostel</a></li>
+												<li><a href="#" title="">WIFI</a></li>
+												<li><a href="#" title="">AC</a></li>
+												<li><a href="#" title="">Camera</a></li>
+												<li><a href="#" title="">Party</a></li>
+												<li><a href="#" title="">Pool</a></li>
+												<li><a href="#" title="">PHP</a></li>
+												<li><a href="#" title="">Offers</a></li>
+												<li><a href="#" title="">Khignt</a></li>
+											</ul>
+										</div><!--widget-tags end-->
+									</div><!--widget end-->
+								</div>
+							</div>
+						</div><!--top-footer end-->
+						<div class="bottom-footer">
+							<div class="copyright">
+								<p>THANKYOU EVERYONE</p>
+							</div><!--copyright end-->
+							<div class="credit">
+								<p>Projecy of : <a href="#" title>Ariel Nazareno, Christian Almazan, LLoyd Hontiveros</a></p>
+							</div><!--credit end-->
+						</div><!--bottom-footer end-->
+					</div>
+				</div>
+			</div>
+		</footer><!--footer end-->
+
+
+
+
+
+	</div><!--wrapper end-->
+
+<!-- Including Jquery Js File -->
+<script type="text/javascript" src="http://creativethemes.us/relax/js/jquery.min.js"></script>

```

Comment:
Pa link naman po ng mga gagamitin nyo na hindi online. Tingin nalang kayo sa "Referenced Frameworks" folder kung meron and kung wla DL nyo nalang at palagay sa "Referenced Frameworks" folder